app/views/admin/invoices/show.html.erb

```diff
@@ -1,10 +1,23 @@
-<h1> Admin Dashboard </h1>
+<nav>
+  <h1 id="admin_title">Admin Dashboard</h1>
+  <ul id="admin_links">
+    <li><%= link_to "Dashboard", admin_index_path %></li>
+    <li><%= link_to "Merchants", admin_merchants_path %></li>
+    <li><%= link_to "Invoices", admin_invoices_path %></li>
+  </ul>
+</nav>
 
+<% statuses = Invoice.statuses.keys %>
 <h2> Invoice ID: <%= @invoice.id %> </h2>
 <p> Status: <%= @invoice.status %> </p>
+<%= form_with model: @invoice, url: admin_invoice_path(@invoice), method: :patch do |form| %>
```

Comment:
Did you know if the binding models approach will work here? I was unable to get it to work on my story and ended up using the URL approach